LayoutTests/fast/js-promise/js-promise-invalid-context-access.html

```diff
@@ -7,7 +7,7 @@
 }
 function f3() {
   var v1 = document.createElement("col");
-  v1.requestPointerLock();
+  v1.requestPointerLock()?.catch(e => {});
```

Comment:
Why silence the exception here and other places?

Reply:
My intent is for the tests to pass whether the PointerLockOptionsEnabled runtime flag is set or not.

If I don't catch the promise rejection, an unhandled promise rejection message will be printed, and the test output will be different and the test will fail. I could add that message to the test expectation, but then the test would fail when PointerLockOptionsEnabled=false.

Reply:
Ok, thanks for clarifying.